src/flag_gems/csrc/cstub.cpp

```diff
@@ -22,6 +22,11 @@ TORCH_LIBRARY(flag_gems, m) {
       "rotary_embedding(Tensor q, Tensor k, Tensor cos, Tensor sin, Tensor? position_ids=None, "
       "bool rotary_interleaved=False) -> (Tensor, Tensor)");  // q and k may be view to other size
   m.def("bmm(Tensor self, Tensor mat2) -> Tensor");
+  // fill operator declaration 
+  m.def("fill_scalar(Tensor input, double value) -> Tensor");
```

Comment:
Be consistent with the function signature, use Scalar for `fill_scalar`.

Also, we suggest 
1. using the same signature as in Aten, since we may also register it for ATen in the future.
2. use marker to mark aliasing, and ! to mark mutation.

`fill.Scalar` and `fill.Tensor` has the same op name with different overload.

```yaml
fill.Scalar(Tensor self, Scalar value) -> Tensor
fill.Tensor(Tensor self, Tensor value) -> Tensor
fill_.Scalar(Tensor(a!) self, Scalar value) -> Tensor(a!)
fill_.Tensor(Tensor(a!) self, Tensor value) -> Tensor(a!)
```